api/src/main/java/org/pmiops/workbench/elasticsearch/ElasticFilters.java

```diff
@@ -44,43 +39,29 @@
   private static final Logger log = Logger.getLogger(ElasticFilters.class.getName());
 
   /** Translates a Cohort Builder search request into an Elasticsearch filter. */
-  public static QueryBuilder fromCohortSearch(CriteriaDao criteriaDao, SearchRequest req) {
-    ElasticFilters f = new ElasticFilters(criteriaDao);
+  public static QueryBuilder fromCohortSearch(CBCriteriaDao cbCriteriaDao, SearchRequest req) {
+    ElasticFilters f = new ElasticFilters(cbCriteriaDao);
     return f.process(req);
   }
 
-  /**
-   * A hand-maintained mapping of which criteria trees map to standard vs source concept IDs. This
-   * indicates which concept ID field we should be querying against for a SearchRequest on a given
-   * tree. TODO(RW-2249): Document or encode this more centrally.
```

Comment:
Does universal search obsolete this ticket? https://precisionmedicineinitiative.atlassian.net/browse/RW-2249

If so, do you mind closing it?

Reply:
Yes, there is a now a standard flag in the criteria table, so now need for the codebase to handle hand maintained source/standard mappings. Closed ticket.